src/css/site.styl

```diff
@@ -84,9 +84,11 @@ input, select
   width: 100vw
   top: 0
   left: 0
-  z-index: 500
+  z-index: 400
   transition: color 300ms
-
+  nav
+    position: relative
+    z-index: 400
```

Comment:
Oh I dunno but my point being to use z-indexes in a stack makes them more understandable to read